packages/azure-http-specs/specs/client/structure/default/client.tsp

```diff
@@ -4,7 +4,7 @@ import "@typespec/spector";
 
 using Spector;
 
-@@scenario(Client.Structure.Service);
+@@`scenario`(Client.Structure.Service);
```

Comment:
@allenjzhang @timotheeguerin it seems the [recent pr](https://github.com/microsoft/typespec/pull/6794/files#diff-8685634079a4b1fdca6f014e6c1a079ab1e786c5689d26a56a16399c0684c793) introduced this changes.

And we have two questions here:
- why only `@@scenario` reported errors? we have a lot of `@scenario` with no errors?
- if we update this way we can't bypass the format, is the fix correct? how to bypass format check?

Reply:
I think we are not going to bypass format.

We will need to bump core first https://github.com/Azure/typespec-azure/pull/2485
(I guess I would actually merge the change in this PR into the above bump core PR)

Reply:
Yeah, do create a manual PR with this change and bump core to have green PR.